src/Cli/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs

```diff
@@ -80,8 +80,29 @@ public void Configure()
             {
                 Stopwatch beforeGenerateAspNetCertificate = Stopwatch.StartNew();
                 GenerateAspNetCertificate();
+
+                if (canShowFirstUseMessages)
+                {
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                    {
+                        // The instructions in this message only apply to Windows and MacOS.
+                        PrintFirstTimeMessageAspNetCertificate();
+                    }
+                    else
```

Comment:
Should this explicitly check for the non-Windows and non-macOS platforms? What if a new platform is added that doesn't support the dev certs at all? Maybe something to do longer term if we have more first run message would be to have a collection and conditionally yield return strings based on the conditions stored with the messages. That would avoid adding more `PrintFirstTimeMessage` methods in the future I think

Reply:
If we end up having more ways to convey this information (more platforms, links, etc.), I'd likely just make a general purpose message with some string replacement tokens (`{1}`) in the resource and have the links sub in as needed. Links don't need localization, so it works pretty well. We did some stuff like that in project system. We can also sub in other resource strings too using the tokens.

Keep in mind, the platforms supported by the actual cert creation only have Windows, MacOS, and *Unix* implementations right now. The non-Windows/non-Mac in this code defaults to Unix, so I just make the same assumption here. 
https://github.com/dotnet/aspnetcore/blob/2fbeeafbafd44d4e44dcc1f676dfcba642a5314e/src/Shared/CertificateGeneration/CertificateManager.cs#L33-L39

And yes, Linux and Unix are different, but our conceptual documentation is specific to Linux. If someone is on a Unix-based platform, I think the Linux docs would be enough to lead them in the right direction to follow.